### PR TITLE
Handle new UV response format

### PR DESCRIPTION
**Pull request by mikepsinn**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /vendor
 /.vagrant/
 /ubuntu-xenial-16.04-cloudimg-console.log
+/.idea/
```
Comment (peter279k) [inline .gitignore:4]: Why this folder is ignored on Git version control?

What tool will generate this folder?

Comment (cmfcmf) [reply .gitignore]: This folder comes from IntelliJ IDEs. I've removed it in 12c2e7d.

Comment (cmfcmf) [reply .gitignore]: (Thank you for reviewing this!)

**Comment by cmfcmf** [comment]
Thank you @mikepsinn! I love how the official website still lists the old format :(

**Comment by mikepsinn** [comment]
@cmfcmf Thanks for all your great work!  :D

**Pull request merged**